docs/content/development/extensions-core/variance.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: doc_page
+---
+
+# Variance aggregator
+
+Used to get variance (or stddev via post aggregator) of numeric column. The algorithm used here is the same with variance udf of apache hive. See javadoc of io.druid.query.aggregation.variance.VarianceHolder for detail.
+
```

Comment:
can we include some docs indicating whether or not for the aggregator to work, the variance/std dev column must be indexed at ingestion time?